Resources/Prototypes/Entities/Structures/Walls/asteroid.yml

```diff
@@ -17,6 +17,10 @@
     mode: NoSprite
   - type: SmoothEdge
   - type: Sprite
+    variants: 2
+    placementVariants:
+    - 1.0
+    - 1.0
```

Comment:
you put this on spritecomp when i think you meant to put it on some kind of floor tile. Sprites don't support anything like this (i'm surprised the linter isn't mad)

Reply:
lol, yea i attempted to do this but it didnt work so i created a spawner with 2 entities and forgot to remove it again.